packages/common/src/contract/path.ts

```diff
@@ -0,0 +1,17 @@
+import path from 'node:path'
+
+export class Path {
+  constructor(private _path: string) {}
+
+  get path(): string {
+    if (path.isAbsolute(this._path)) {
+      return this._path
+    } else {
+      return path.resolve(process.cwd(), this._path)
```

Comment:
It requires users to run the command in the root directory. What if using something like https://github.com/ckb-js/kuai/blob/8e3cdb59fca7eddabeb592d8d169872d62e21919/packages/core/src/util/packageInfo.ts#L19